src/sentry/integrations/github/utils.py

```diff
@@ -18,7 +18,8 @@ def get_jwt():
         # JWT expiration time (10 minute maximum)
         'exp': exp,
         # Integration's GitHub identifier
-        'iss': options.get('github-app.id'),
+        'iss': github_id or options.get('github-app.id'),
```

Comment:
Does doing it like this actually work? I did something like this recently and ended up with a boolean value.

Reply:
```python
>>> False or 'string'
'string'
>>> None or 'string'
'string'
>>> 0 or 'string'
'string'
```

Reply:
this worked when testing locally - if it returned a boolean value for the iss then the jwt definitely wouldn't have worked so I _think_ this is correct